db/seeds.rb

```diff
@@ -5,3 +5,4 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
```

Comment:
Looks cool! Please show me how to do this ;)

Reply:
this was generated for us but all it is is calling a create method on an existing model :D

Reply:
In order to actually add the user to that database you need to run 
```
$ rails db:seed
```

in your terminal